src/Mint.js

```diff
@@ -0,0 +1,354 @@
+import { Fragment, useState } from 'react';
+import Web3 from "web3";
+import { ExclamationCircleIcon, XIcon } from '@heroicons/react/solid';
+import { Transition } from '@headlessui/react';
+
+import gif from './assets/148-167_40_04.gif';
+import TheFacesFund from './abi/TheFacesFund.json';
+
+const defaultErrorTitle = 'Whoops!';
```

Comment:
nit: "whoops!" sounds a little aloof, maybe just omit this and say "Looks like something went wrong!" ?

Reply:
Yeah that makes sense. It also shortens the other messages